udm/intermediate/shopify-samples/orders-json/Backorder BOPIS - content.json

```diff
@@ -0,0 +1,391 @@
+{
+  "order": {
+    "id": 5524199899389,
+    "admin_graphql_api_id": "gid://shopify/Order/5524199899389",
+    "app_id": 580111,
+    "browser_ip": "111.118.250.178",
+    "buyer_accepts_marketing": false,
+    "cancel_reason": null,
+    "cancelled_at": null,
+    "cart_token": "c1-205acf291edff427599b826e4816ad86",
```

Comment:
empty the value

Reply:
@sanjanashrivastava Removed the cart token value from all the JSON.